probe/ebpf/flow.c

```diff
@@ -85,6 +85,7 @@ static inline void fill_payload(struct __sk_buff *skb, int offset, struct flow *
 static inline void fill_transport(struct __sk_buff *skb, __u8 protocol, int offset,
 	struct flow *flow, int len)
 {
+	__u64 tm = bpf_ktime_get_ns();
```

Comment:
get time only when needed (in TCP only)
Would be nice to have this feature as configurable (disabled by default)

Reply:
will change this